spec/classes/agent_spec.rb

```diff
@@ -58,11 +58,12 @@
 
         it {
           should contain_file('sensu_agent_config').with({
-            'ensure'  => 'file',
-            'path'    => '/etc/sensu/agent.yml',
-            'content' => agent_content,
-            'require' => 'Package[sensu-go-agent]',
-            'notify'  => 'Service[sensu-agent]',
+            'ensure'    => 'file',
+            'path'      => '/etc/sensu/agent.yml',
+            'content'   => agent_content,
+            'show_diff' => 'false',
```

Comment:
why the `show_diff` set to `false`. This is very useful to see what is changing.

Reply:
@treydock could you please comment on this? The `show_diff` question is the only one outstanding before merge.

Reply:
Because there is a password config option in `agent.yml`.  A default install of sensu-go adds the agent user with same default password as admin user and thought if someone is going to change admin user they might be also be changing agent user password which is now possible with this pull request.  The password for agent user has to be updated in `agent.yml` for all agents if agent user password is changed.

Reply:
Please remove from here and in another PR let's make it a param that can be set and apply it to all config files.

Reply:
Removed `show_diff` change.